src/components/Dropdown/dropdown.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './dropdown.scss';
+
+class Dropdown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      expanded: false,
+      selectedID: 0,
+      options: props.options,
+    };
+    this.toggleDropdown = this.toggleDropdown.bind(this);
+    this.selectOption = this.selectOption.bind(this);
+  }
+
+  selectOption(e) {
+    const option = e.target;
+    this.setState({
+      selectedID: option.getAttribute('selectednum'),
+    });
+  }
+
+  toggleDropdown() {
+    this.setState( {
+      expanded: !this.state.expanded,
+    });
+  }
+
+  renderMenuItems() {
+    return this.state.options.map((opt, i) => (
+      <li
+        className="dropdown__items__element"
+        key={i.toString()}
```

Comment:
don't use array index in keys

Reply:
there are a lot of article why not to do that, e.g. one of them https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318